backend/src/routes/users.js

```diff
@@ -1,15 +1,92 @@
+const auth = require('../auth');
 const express = require('express');
 const router = express.Router();
 const models = require('../models');
+const db = require('../db');
+const utils = require('../utils');
+const randtoken = require('rand-token');
 
-router.get('/:id', (req, res) => {
-    res.send('returning specific user ' + req.params.id);
-});
+const getUser = async (req, res) => {
+    try {
+        const phoneNumber = req.body.phoneNumber;
+        const user = await db.users.getUser(phoneNumber);
+        res.status(200).send(user);
+    } catch (err) {
+        res.status(420).send('Failed to get user: ' + err);
+    }
+};
 
-router.get('/', (req, res) => {
-    models.users.findAll({}).then(function(users) {
+const getAllUsers = async (req, res) => {
+    try {
+        const users = await db.users.getAllUsers();
         res.status(200).send(users);
-    });
-});
+    } catch (err) {
+        res.status(420).send('Failed to get user list: ' + err);
+    }
+};
+
+const updateUser = async (req, res) => {
```

Comment:
This method allows updating any other users stuff. Should be based on the `req.user` too.